systemtests/tests/testfind/etc/bareos/bareos-dir.d/fileset/SelfTest.conf.in

```diff
@@ -14,7 +14,7 @@ FileSet {
       fstype = zfs
       fstype = btrfs
     }
-   File = "@sbindir@"
+   File = "@tmpdir@/data"
    #File=<@tmpdir@/file-list
```

Comment:
Is it possible to get the originam mechanism (line 18) also working?

Reply:
I can easily add this but it has some  issues linked to it 
I had a meeting once with Stephan and aala only i think when you were on holidays and we discussed wether i should keep debugging the `addFileToFileset` or leave it out since it is a nice to have the guys recommended to leave it and advance with the pull request for that moment but all the ground work is set for it and it's pretty nice to add so i will do that now